pkg/scheduler/schedule_one_test.go

```diff
@@ -2227,7 +2227,7 @@ func TestSchedulerSchedulePod(t *testing.T) {
 			nodes:              []string{"node1", "node2", "node3"},
 			pod:                st.MakePod().Name("test-prefilter").UID("test-prefilter").Obj(),
 			wantNodes:          sets.New("node2"),
-			wantEvaluatedNodes: ptr.To[int32](1),
+			wantEvaluatedNodes: ptr.To[int32](3),
```

Comment:
Maybe this number shouldn't have changed?

Although it's only useful for debugging.

Reply:
It should be changed.

Please refer to the change in pkg/scheduler/scheduler.go in this PR: After this PR, EvaluatedNodes contains the number of nodes that filtered out by PreFilterResult.
https://github.com/kubernetes/kubernetes/pull/119779/files#r1439161728

Reply:
Yes, I understand that it's not a bug. However, it was useful to have the value 1 to know that only one node passed PreFilter.